libs/platform/src/lib/table/table.component.html

```diff
@@ -338,21 +366,27 @@
                             <td
                                 *ngFor="let column of _visibleColumns; let colIdx = index"
                                 fd-table-cell
-                                [fdpTableCellResizable]="_getColumnResizableSide(colIdx)"
-                                [columnName]="column.name"
-                                role="cell"
+                                role="gridcell"
+                                [focusable]="true"
+                                [fd-table-cell-selectable]="_getCellNavigationId(rowIdx + 1, colIdx)"
                                 [headers]="id + '__' + column.name"
+                                [fdp-table-cell-resizable]="_getColumnResizableSide(colIdx)"
+                                [columnName]="column.name"
                                 [style.textAlign]="column.align"
                                 [class.fd-table__cell--fixed]="_freezableColumns?.includes(column.name)"
                                 [class.fd-table__cell--fixed-last]="column.name === freezeColumnsTo"
                                 [class.fd-table__cell--expand]="_isTreeRowFirstCell(colIdx, row)"
                                 [ngStyle]="_getCellStyles(column)"
                                 [attr.aria-expanded]="_isTreeRow(row) ? row.expanded : null"
+                                attr.aria-label="Row {{ rowIdx }} of {{ _totalItems }}, Column {{ colIdx }} of {{
+                                    _tableColumnsLength
+                                }}. "
```

Comment:
shouldn't we add i18n marker?

Reply:
I would also suggest to move it to getter.

Reply:
Added